docs/docs/getting-started/cluster-setup.md

```diff
@@ -86,7 +86,10 @@ az group create \
 
 ## Create AKS cluster
 
-First create an AKS cluster:
+First create an AKS cluster. We can't directly create a CoCo-only cluster, so we'll need to create a
```

Comment:
Clusters are never "CoCo-only".

Reply:
Would you mind to elaborate?

Reply:
Well, you can always deploy non-confidential workloads on nodes that also enable coco workloads.